src/main/java/org/openstreetmap/atlas/geography/atlas/change/description/descriptors/ChangeDescriptorComparator.java

```diff
@@ -53,23 +54,72 @@ private int complexCompare(final ChangeDescriptor left, final ChangeDescriptor r
     private int genericSetChangeCompare(final GenericElementChangeDescriptor left,
             final GenericElementChangeDescriptor right)
     {
-        if (!left.getDescription().equals(right.getDescription()))
+        if (!left.getName().equals(right.getName()))
```

Comment:
Here the name is an enum like above, right?
```suggestion
        if (left.getName() != right.getName())
```